#____________________________________________________________________________________________
# servoing_posx
#____________________________________________________________________________________________
# --

---
float64[] servoing_posx
# servoing_posx : 서보잉 자세
float64[] vertices
# vertices : 원래 8개 물체의 좌표
